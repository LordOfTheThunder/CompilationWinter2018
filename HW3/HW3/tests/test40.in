struct potato {
	int a;
	bool b;
	byte c;
	int d;
};
int PutToPotato(int a, bool b, byte c, int d, struct potato pot) {
	pot.a = a;
	pot.b = b;
	pot.c = c;
	pot.d = d;
}

void callPutPotato() {
	struct potato pot;
	PutToPotato(313, false, 33b, 22b, pot);
}
