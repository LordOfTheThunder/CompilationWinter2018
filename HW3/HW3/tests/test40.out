---end scope---
a INT -8
b BOOL -7
c BYTE -6
d INT -5
pot struct potato -4
---end scope---
pot struct potato 0
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
struct potato {INT a,BOOL b,BYTE c,INT d}
PutToPotato (INT,BOOL,BYTE,INT,potato)->INT 3
callPutPotato ()->VOID 4
